.github/workflows/villagesql_flavor_test.yml

```
@@ -0,0 +1,145 @@
+  VILLAGESQL_SHA256: "8b15522a973b17b430ed9e64e8bdbf97bf858cef028bfbc7f9b9608002406393"
+  VILLAGESQL_TARBALL: "villagesql-dev-server-0.0.3-dev-linux-x86_64.tar.gz"
+  VILLAGESQL_URL: "https://github.com/villagesql/villagesql-server/releases/download/0.0.3/villagesql-dev-server-0.0.3-dev-linux-x86_64.tar.gz"
+  # VillageSQL uses its own version scheme (0.0.3). Map to MySQL 8.0.40 for
+  # capability lookups (mysqld --initialize, CREATE USER, GTID, etc.)
+  MYSQL_MAPPED_VERSION: "8.0.40"
+  GO111MODULE: on
+  SANDBOX_BINARY: ${{ github.workspace }}/opt/mysql
+
+jobs:
+  villagesql-deploy:
+    name: Deploy (VillageSQL ${{ env.VILLAGESQL_VERSION }})
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)